Backpack/Spinner/Classes/BPKSpinner.h

```diff
@@ -32,8 +32,9 @@ typedef NS_ENUM(NSUInteger, BPKSpinnerSize) {
  */
 typedef NS_ENUM(NSUInteger, BPKSpinnerStyle) {
     BPKSpinnerStylePrimary = 0,
-    BPKSpinnerStyleDark = 1,
-    BPKSpinnerStyleLight = 2,
+    BPKSpinnerStyleSecondary = 1,
```

Comment:
This is breaking because the underlying value of `BPKSpinneStyleDark` and `BPKSpinnerStyleLight` have both changed. Can we just add `BPKSpinnerStyleSecondary` as with the value `3` instead?